src/server/pom.xml

```diff
@@ -49,14 +50,15 @@
         </cucumber.no-versions-to-test>
         <cucumber.upgrade-versions>${cucumber.no-versions-to-test}</cucumber.upgrade-versions>
         <cxf-version>3.4.5</cxf-version>
+        <shiro-version>1.8.0</shiro-version>
     </properties>
 
     <dependencies>
         <!--compile scope -->
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>23.0</version>
+            <version>24.1.1-jre</version>
```

Comment:
This is unfortunately the last version that still has the old `Futures.addCallback` method without a 3rd argument (the executor), which seems used in Reaper codebase. We might want to upgrade to more recent versions of Guava later on (Guava is a frequent source of CVEs), but that would require code changes.